### ISSUE TITLE
Multiple tapouts 

### ISSUE THREAD

**Issue by TechSupportJosh** [netkit-jh/netkit-jh-build#31]
**Describe the solution you'd like**
Allow the ability to have multiple tapouts within a single lab. 

This means you can do something such as:
machineA[0]=tap,10.0.0.1,10.0.0.2
machineB[0]=tap,20.0.0.1,20.0.0.2

and both machine A and B will be able to ping out to the internet independently.

This will require modifying any nk_tap_ instances and introducing a unique identifier for each one, as well as intelligent directing of the packets. This may be able to be handled by iptables MASQUERADE.

**Comment by SylinsicWMG** [comment]
Multiple machines can currently use tap interfaces, so long as they are all on the same subnet, with the host IP being the same.
i.e.
```
machineA[0]=tap,10.0.0.1,10.0.0.2
machineB[0]=tap,10.0.0.1,10.0.0.3
```
Only a singular interface is currently created on the host machine which would be shared by all tap's provided the usecase above. If different IP's are used however, there aren't multiple interfaces, and hence only the first machine brought up will have a valid connection

**Comment by AdamBromiley** [comment]
Issue has been resolved in version 2.0.0-beta. Tap interfaces can coexist so long as they have unique IP addresses on /24 subnets that don't overlap.
